x-pack/plugins/watcher/public/components/delete_watches_modal.tsx

```diff
@@ -58,8 +58,8 @@ export const DeleteWatchesModal = ({
                 'xpack.watcher.sections.watchList.deleteSelectedWatchesSuccessNotification.descriptionText',
                 {
                   defaultMessage:
-                    'Deleted {numWatchesToDelete, plural, one {watch} other {{numSuccesses} out of # selected watches}} ',
-                  values: { numSuccesses, numWatchesToDelete },
+                    'Successfuly deleted {numSuccesses, number} {numSuccesses, plural, one {watch} other {watches}}',
```

Comment:
I think we can drop "Successfully". I know we use this in Index Management but I don't see it anywhere else. Feel free to confirm this with Gail.